packages/seacas/applications/blot/cgi/vdicps.f

```diff
@@ -1801,8 +1801,9 @@ SUBROUTINE VDIQCO(NUM,INDEX,CLRARY,CLRMOD)
             CALL VBERRH(724,5)
             GOTO 100
          END IF
-           DO 200 IC=1,3
-  200      CLRARY(IC,I)=PCOLS(IC,INDEXN)
+           CLRARY(1,I)=PCOLS(1,INDEXN)
```

Comment:
`CLRARY(1:3,I)=PCOLS(1:3,INDEXN)` ?

Reply:
That would work, but I usually stick with fortran-77 in the seacas portions... No real reason other than inertia.